src/components/Navigation/Footer.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import withStyles from 'isomorphic-style-loader/lib/withStyles';
+import Grid from 'react-bootstrap/lib/Grid';
+import Row from 'react-bootstrap/lib/Row';
+
+import styles from './Footer.scss';
+
+const Footer = React.createClass({
+
+  render() {
+    const url = [
+      'https://github.com/kodeklubben/oppgaver/',
+      'http://kidsakoder.no/',
+      'https://github.com/kodeklubben/kodeklubben.github.io/archive/master.zip',
+      'http://kidsakoder.no/2015/07/03/kodeklubben-trondheim-utvikler-materiell-i-sommer/'
+    ];
+
+    return (
+      <Grid fluid={true} className={styles.container}>
+        <Row className={styles.center}>
+          <p>Bidra?</p>
```

Comment:
Er i større grad det. Men er det lurt å skjule link til repoet når default modus er elev? Tenker denne linken gjelder for alle selv om det sannsynligvis er flest lærere/elever som er interessert i å bidra. Hva tenker dere @kristandre @NorwegianKiwi?